### ISSUE TITLE
04 - Retrain with all data and make 10 year forecast

### ISSUE THREAD

**Issue by john-hawkins** [pinglainstitute/energy-gdp-emissions#4]
The final step is to train a final model using all of the data, use the hyper-parameter settings from your best model.
--- This means including the final 9 years of test data into the training data.
--- There is no test set for this model, which is why we fix all hyper-parameters.

Use this model to make a forecast for 10 years forward from the last data points in the training data. 
Plot the forecast for each of the 3 countries.
